### ISSUE TITLE
Error when working with fnm

### ISSUE THREAD

**Issue by lirantal** [lirantal/npq#241]
### Discussed in https://github.com/lirantal/npq/discussions/239

<div type='discussions-op-text'>

<sup>Originally posted by **baruchiro** December 27, 2022</sup>
Documentation of my process when I upgraded `npq` to the latest version, and received error on the next command.

### System
```
OS: Windows (WSL)
fnm 1.31.1
```

### Run
```shell
> yarn test
...
...
   ╭──────────────────────────────────────╮
   │                                      │
   │   Update available 2.0.23 → 2.1.0    │
   │      Run npm i -g npq to update      │
   │                                      │
   ╰──────────────────────────────────────╯

> npm i -g npq
...
...

> yarn test
/home/baruch/.fnm/node-versions/v16.18.1/installation/lib/node_modules/npq/lib/helpers/cliSupportHandler.js:7
const updateNotifier = require('update-notifier')
                       ^

Error [ERR_REQUIRE_ESM]: require() of ES Module /home/baruch/.fnm/node-versions/v16.18.1/installation/lib/node_modules/npq/node_modules/update-notifier/index.js from /home/baruch/.fnm/node-versions/v16.18.1/installation/lib/node_modules/npq/lib/helpers/cliSupportHandler.js not supported.
Instead change the require of index.js in /home/baruch/.fnm/node-versions/v16.18.1/installation/lib/node_modules/npq/lib/helpers/cliSupportHandler.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/home/baruch/.fnm/node-versions/v16.18.1/installation/lib/node_modules/npq/lib/helpers/cliSupportHandler.js:7:24)
    at Object.<anonymous> (/home/baruch/.fnm/node-versions/v16.18.1/installation/lib/node_modules/npq/bin/npq-hero.js:5:20) {
  code: 'ERR_REQUIRE_ESM'
}

> fnm use
Using Node v18.12.1

> yarn test
/home/baruch/.fnm/node-versions/v16.18.1/installation/lib/node_modules/npq/lib/helpers/cliSupportHandler.js:7
const updateNotifier = require('update-notifier')
                       ^

Error [ERR_REQUIRE_ESM]: require() of ES Module /home/baruch/.fnm/node-versions/v16.18.1/installation/lib/node_modules/npq/node_modules/update-notifier/index.js from /home/baruch/.fnm/node-versions/v16.18.1/installation/lib/node_modules/npq/lib/helpers/cliSupportHandler.js not supported.
Instead change the require of index.js in /home/baruch/.fnm/node-versions/v16.18.1/installation/lib/node_modules/npq/lib/helpers/cliSupportHandler.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/home/baruch/.fnm/node-versions/v16.18.1/installation/lib/node_modules/npq/lib/helpers/cliSupportHandler.js:7:24)
    at Object.<anonymous> (/home/baruch/.fnm/node-versions/v16.18.1/installation/lib/node_modules/npq/bin/npq-hero.js:5:20) {
  code: 'ERR_REQUIRE_ESM'
}

Node.js v18.12.1
```

- Why `fnm` is using `v18.12.1`, but the error is from `v16.18.1`?</div>

**Comment by lirantal** [comment]
@baruchiro can you check if npq is fixed with the latest version npq@2.1.1 ?

**Comment by baruchiro** [comment]
Yes, it fixed!!
